p2p/lightchain.py

```diff
@@ -95,12 +108,17 @@ def register_peer(self, peer: BasePeer) -> None:
                 # Either the peer disconnected or our cancel_token has been triggered, so break
                 # out of the loop to stop attempting to sync with this peer.
                 break
-            # We currently implement only the LES commands for retrieving data (apart from
-            # Announce), and those should always come as a response to requests we make so will be
-            # handled by LESPeer.handle_sub_proto_msg().
+
             if isinstance(cmd, les.Announce):
                 peer.head_info = cmd.as_head_info(msg)
                 self._announcement_queue.put_nowait((peer, peer.head_info))
+            elif isinstance(msg, dict):
+                request_id = msg.get('request_id')
+                if request_id is not None and request_id in self._pending_replies:
```

Comment:
Can we document the case where `request_id == None` so it's clear why we silently do nothing in that case

Reply:
Sure, it's because not all LES messages include a request_id. I've added a comment explaining